projects/vdk-plugins/vdk-trino/src/vdk/plugin/trino/ingest_to_trino.py

```diff
@@ -91,6 +91,8 @@ def _ingest_payload(
 
     @staticmethod
     def __to_bool(value: str) -> bool:
```

Comment:
I know that arguments type is not enforced in Python, but do we need the str type (does it help us somehow), since now we can also accept bool?

Reply:
I will fix it. It's not enforeced by IDE shows warnings if there's type mismatch so it's important to be accurate.